client/src/components/LocalAsc/LocalAsc.tsx

```diff
@@ -169,6 +171,14 @@ export enum AlertColor {
     SUCCESS = 'success'
 }
 
+export const ExternalLinkRenderer = props => {
```

Comment:
We hebben al een ExternalLink component in /components. Beter om deze te gebruiken?